homeworks/rostyslav.vyshemirskyi_Rooosss/PokemonProject/scripts/pokemons.js

```diff
@@ -0,0 +1,326 @@
+            'Rock',
+        ],
+        number: '0006',
+        height: 1102.0,
+        collectibles_slug: 'charizard',
+        featured: 'true',
+        slug: 'charizard',
+        name: 'Charizard',
+        ThumbnailAltText: 'Charizard',
+        ThumbnailImage: 'https://assets.pokemon.com/assets/cms2/img/pokedex/detail/006.png',
+        id: 6,
+        type: [
+            'fire',
+            'flying',
+        ],
+    },
+    {
+        abilities: [
+            'Drought',
+        ],
+        detailPageURL: '/us/pokedex/charizard',
+        weight: 221.6,
+        weakness: [
+            'Water',
+            'Electric',
+            'Rock',
+        ],
+        number: '0006',
+        height: 67.0,
+        collectibles_slug: 'charizard',
+        featured: 'true',
+        slug: 'charizard',
+        name: 'Charizard',
+        ThumbnailAltText: 'Charizard',
+        ThumbnailImage: 'https://assets.pokemon.com/assets/cms2/img/pokedex/detail/006.png',
+        id: 6,
+        type: [
+            'fire',
+            'flying',
+        ],
+    },
+    {
+        abilities: [
+            'Tough Claws',
+        ],
+        detailPageURL: '/us/pokedex/charizard',
+        weight: 243.6,
+        weakness: [
+            'Ground',
+            'Rock',
+            'Dragon',
+        ],
+        number: '0006',
+        height: 67.0,
+        collectibles_slug: 'charizard',
+        featured: 'true',
+        slug: 'charizard',
+        name: 'Charizard',
+        ThumbnailAltText: 'Charizard',
+        ThumbnailImage: 'https://assets.pokemon.com/assets/cms2/img/pokedex/detail/006.png',
+        id: 6,
+        type: [
+            'fire',
+            'dragon',
+        ],
+    },
+];
+
+const cardsList = document.querySelector('[data-type="cards-list"]');
+
+function createCardElement(pokemonsList) {
+    const cardItem = document.querySelector('[data-type="card-template"]');
+    const cardContent = cardItem.content;
+    const cardName = cardContent.querySelector('[data-card-name]');
+    const cardImage = cardContent.querySelector('[data-card-image]');
+    const cardHeight = cardContent.querySelector('[data-card-height]');
+    const cardWeight = cardContent.querySelector('[data-card-weight]');
+    const cardNumber = cardContent.querySelector('[data-card-number]');
+
+    cardImage.src = pokemonsList.ThumbnailImage;
+    cardName.textContent = pokemonsList.name;
+    cardHeight.textContent = pokemonsList.height;
+    cardWeight.textContent = pokemonsList.weight;
+    cardNumber.textContent = pokemonsList.number;
+    return cardContent.cloneNode(true);
+}
+
+function renderCards(container, pokemonsList) {
+    container.innerHTML = '';
+    const fragment = new DocumentFragment();
+    pokemonsList.forEach((pokemon) => {
+        const card = createCardElement(pokemon);
+        fragment.appendChild(card);
+    });
+    container.appendChild(fragment);
+}
+
+const formFilters = document.querySelector('[data-type="filters"]');
```

Comment:
Hey, @Rooosss, the common approach is to group the variables declaration on top, and declare functions below:
```js
const pokemons = ...
const cardsList =...
const formFilters =...
... // all variables
function createCardElement(pokemonsList) {...}
.... // all functions


```